.buildkite/pipeline_jax_tpu7x.yml

```diff
@@ -0,0 +1,311 @@
+         if [[ "$$NIGHTLY" == "1" ]]; then
+           .buildkite/scripts/run_in_docker.sh bash /workspace/tpu_inference/tests/e2e/benchmarking/mlperf.sh
+         else
+           echo "Skipping: NIGHTLY environment variable not set"
+           exit 0
+         fi
+
+   - label: "TPU7x E2E MLPerf tests for JAX + vLLM models on single chip"
+     key: tpu7x_test_3
+     soft_fail: true
+     env:
+       MODEL_IMPL_TYPE: "vllm"
+       IS_FOR_V7X: "true"
+     agents:
+       queue: tpu_v7x_2_queue
+     commands:
+       - .buildkite/scripts/run_in_docker.sh bash /workspace/tpu_inference/tests/e2e/benchmarking/mlperf.sh
+
+   - label: "TPU7x E2E MLperf tests for Llama4 models"
+     key: tpu7x_test_4
+     soft_fail: true
+     env:
+       NEW_MODEL_DESIGN: "1"
+       USE_V6E8_QUEUE: "True"
+       IS_FOR_V7X: "true"
+     agents:
+       queue: tpu_v7x_8_queue
+     commands:
+       - |
+         if [[ "$$NIGHTLY" == "1" ]]; then
+           .buildkite/scripts/run_in_docker.sh bash /workspace/tpu_inference/tests/e2e/benchmarking/mlperf.sh
+         else
+           echo "Skipping: NIGHTLY environment variable not set"
+           exit 0
+         fi
+
+
+   - label: "TPU7x E2E multi modality test"
+     key: tpu7x_test_5
+     soft_fail: true
+     env:
+      IS_FOR_V7X: "true"
+     agents:
+       queue: tpu_v7x_2_queue
+     commands:
+       - |
+         if [[ "$$NIGHTLY" == "1" ]]; then
+           .buildkite/scripts/run_in_docker.sh \
+             bash -c 'python3 -m pytest -s -v -x /workspace/tpu_inference/tests/e2e/test_multi_modal_inference.py && \
+              bash /workspace/tpu_inference/tests/e2e/benchmarking/mm_bench.sh'
+         else
+           echo "Skipping: NIGHTLY environment variable not set"
+           exit 0
+         fi
+
+   - label: "TPU7x E2E speculative decoding test"
+     key: tpu7x_test_6
+     soft_fail: true
+     env:
+      IS_FOR_V7X: "true"
+     agents:
+       queue: tpu_v7x_2_queue
+     commands:
+       - |
+         if [[ "$$NIGHTLY" == "1" ]]; then
+           .buildkite/scripts/run_in_docker.sh \
+             bash -c 'python3 -m pytest -s -v -x /workspace/tpu_inference/tests/e2e/test_speculative_decoding.py'
+         else
+           echo "Skipping: NIGHTLY environment variable not set"
+           exit 0
+         fi
+
+   - label: "TPU7x JAX unit tests"
+     key: tpu7x_test_7
+     soft_fail: true
+     env:
+      IS_FOR_V7X: "true"
+     agents:
+       queue: tpu_v7x_2_queue
+     commands:
+       - |
+         IS_FOR_V7X=true .buildkite/scripts/run_in_docker.sh \
+           python3 -m pytest -s -v -x /workspace/tpu_inference/tests/ \
+           --ignore=/workspace/tpu_inference/tests/kernels \
+           --ignore=/workspace/tpu_inference/tests/lora \
+           --ignore=/workspace/tpu_inference/tests/e2e \
+           --ignore=/workspace/tpu_inference/tpu_inference/mock \
+           --ignore=/workspace/tpu_inference/tests/layers/vllm/test_compressed_tensors_moe.py \
```

Comment:
@QiliangCui Could you enable this /workspace/tpu_inference/tests/layers/vllm/test_compressed_tensors_moe.py  test? It was disabled previously because it can only be run on v7.